conans/model/conan_file.py

```diff
@@ -387,7 +387,7 @@ def package_info(self):
     def run(self, command, output=True, cwd=None, win_bash=False, subsystem=None, msys_mingw=True,
             ignore_errors=False, run_environment=False, with_login=True, env="conanbuildenv"):
 
-        command = environment_wrap_command(env, command)
+        command = environment_wrap_command(env, command, cwd=self.generators_folder)
```

Comment:
Now the wraps will have absolute paths to the generators folder because sometimes you have to run the command inside (cwd) a different folder.